src/core/include/openvino/op/squeeze.hpp

```diff
@@ -4,38 +4,75 @@
 
 #pragma once
 
-#include "openvino/op/op.hpp"
+#include <functional>
+#include <utility>
+
+#include "openvino/op/util/squeeze_base.hpp"
 
 namespace ov {
 namespace op {
 namespace v0 {
 /// \brief Squeeze operation.
 ///
 /// \ingroup ov_ops_cpp_api
-class OPENVINO_API Squeeze : public Op {
+class OPENVINO_API Squeeze : public util::SqueezeBase {
 public:
     OPENVINO_OP("Squeeze", "opset1");
 
     Squeeze();
+    /// \brief Constructs a squeeze v0 operation.
+    ///
+    /// \param data Input tensor with data
+    Squeeze(const Output<Node>& data);
+    /// \brief Constructs a squeeze v0 operation.
+    ///
+    /// \param data Input tensor with data
+    /// \param axis The axis along which to squeeze the input tensor.
```

Comment:
```suggestion
    /// \param axes The axes along which to squeeze the input tensor.
```
axis - is just one, axes - can be multiple